Library/Homebrew/utils/analytics.sh

```diff
@@ -0,0 +1,71 @@
+setup-analytics() {
+  if [[ -z "$HOMEBREW_ANALYTICS" || -n "$HOMEBREW_NO_ANALYTICS" ]]
+  then
+    return
+  fi
```

Comment:
In a recently merged PR you used a different style (see [this code](https://github.com/Homebrew/brew/blob/4a7cd160c31969b79979ae9bd20afcf82d9513c0/Library/brew.sh#L234-L235) or below) and I like it more (because it's closer to its Ruby counterpart). Maybe change this to make them more alike?

```shell
[[ -n "$HOMEBREW_AUTO_UPDATE" ]] || return
[[ -z "$HOMEBREW_NO_AUTO_UPDATE" ]] || return
```